include/NovelRT/Graphics/Camera.h

```diff
@@ -0,0 +1,61 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_GRAPHICS_CAMERA_H
+#define NOVELRT_GRAPHICS_CAMERA_H
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Graphics {
+  class Camera {
+
+    NOVELRT_EVENT(CameraViewChanged, CameraViewChangedEventArgs)
+
+  //TODO: Look Im just trying to make this legal code okay?
+  public:
+    enum class CameraMode : unsigned int {
```

Comment:
nit: this should probably be its own enum
I also don't think we need an enum to differentiate
instead, we probably just need `static unique_ptr<Camera> createPerspective` and `createOrthographic`.

Reply:
Should I delete the enum then?

Reply:
Enum deleted. I'll look into implementing these. :)

Reply:
I'm making a ticket for the perspective view. Orthographic has been implemented. Resolving.